src/adagrams.js

```diff
@@ -1,7 +1,107 @@
+const letterPool = {
+  A: [9, 1],	
+  B: [2, 3],
+  C: [2, 3],	
+  D: [4, 2],	
+  E: [12, 1],	
+  F: [2, 4],
+  G: [3, 2],
+  H: [2, 4],	
+  I: [9, 1],
+  J: [1, 8],	
+  K: [1, 5],	
+  L: [4, 1],	
+  M: [2, 3],	
+  N: [6, 1],
+  O: [8, 1],
+  P: [2, 3],
+  Q: [1, 10],
+  R: [6, 1],
+  S: [4, 1],
+  T: [6, 1],
+  U: [4, 1],
+  V: [2, 4],
+  W: [2, 4],
+  X: [1, 8],
+  Y: [2, 4],
+  Z: [1, 10]
+}
+
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const letters = [];
+    for (const letter in letterPool) {
+      let i = 0
+      do {
+        letters.push(letter);
+        i += 1
+      } while (i < letterPool[letter][0]);
+    }
+
+    letters.sort(() => Math.random() - 0.5);
+
+    let hand = letters.slice(0,10);
+    return hand;
+  },
+
+  usesAvailableLetters(word, hand = drawLetters()) {
+    // if letter is found in hand, the search continues at an incremented index. this way a letter in hand cant be counted twice
+    return word.toUpperCase().split('').every(( index => letter => index = hand.indexOf(letter, index) + 1)(0));
```

Comment:
Wow. This is a lot on one line. Looks slick, but at the same time, consider splitting up this logic to enhance readability.